Sources/Clappr_iOS/Classes/Plugin/Core/MediaControl/MediaControl.swift

```diff
@@ -212,6 +219,20 @@ open class MediaControl: UICorePlugin, UIGestureRecognizerDelegate {
         }
     }
 
+    private func loadDefaultPlugins() {
```

Comment:
Hey @brunovpl ,
As I commented in [PlayButton Plugin request](https://github.com/clappr/clappr-ios/pull/230#discussion_r225717983), according to Clappr architecture, we should not do this here.

Take a look into the other comment for more context.

Reply:
hey @nabilsafatli , this way to load Plugins is only a temp way to be able to implement and test each media control plugin task. Once we have the loader task ready we would be able to remove this "temp way" and make it right; we already have a [card](https://trello.com/c/qIwhSI3N/382-plugins-de-3-load-de-plugins-de-mediacontrol-no-loader) for this. = )